### PR TITLE
♻️ Build all existing 3p vendors integrations file

### PR DESCRIPTION
**Pull request by powerivq**
Add all existing 3p vendors.

### CODE REVIEW

**Review by calebcordry — CHANGES_REQUESTED** [review]
Looks like the export name might be wrong for weborama?

**Comment by powerivq** [comment]
> Looks like the export name might be wrong for weborama?

https://github.com/ampproject/amphtml/blob/3027b8e0c2a35f4fffac2f9a56274c0215758c76/3p/integration.js#L558 its vendor type name is weborama-display.

File: 3p/vendors/weborama-display.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * limitations under the License.
+ */
+
+// src/polyfills.js must be the first import.
+import '../polyfills';
+
+import {draw3p, init} from '../integration-lib';
+import {register} from '../3p';
+
+import {weborama} from '../../ads/vendors/weborama';
```
Comment (calebcordry) [inline 3p/vendors/weborama-display.js:23]: This is the line breaking on travis.

**Comment by powerivq** [comment]
OO yeah you're right.

**Review by calebcordry — APPROVED** [review]

**Review by lannka — APPROVED** [review]

**Comment by powerivq** [comment]
@rsimha Needs your owner approval on `build-system/tasks/dist.js` change. It prevents building 3p vendors when `core_runtime_only` is set.

**Review by danielrozenberg — APPROVED** [review]
Approval for build-system change

**Comment by rsimha** [comment]
@powerivq @calebcordry @lannka This PR has increased the duration of `gulp dist` by a factor of 2.5x (~4m -> ~10m).

Since this affects our local development, CI, and release workflows, can we revert until there's a discussion about the purpose of this change? There may be other ways build 3p vendor files without affecting the build time for the runtime.

**Pull request merged**